tools/tar2idx.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
I suggest remove the github reference for "imgtorec.py" and add the script file here